spec/requests/user/user_profile_spec.rb

```diff
@@ -62,18 +62,6 @@
       expect(response.body).not_to include("<meta name=\"googlebot\" content=\"noindex\">")
     end
 
-    it "renders rss feed link if any stories" do
-      create(:article, user_id: user.id)
-
-      get "/#{user.username}"
-      expect(response.body).to include("/feed/#{user.username}")
-    end
-
-    it "does not render feed link if no stories" do
-      get "/#{user.username}"
-      expect(response.body).not_to include("/feed/#{user.username}")
```

Comment:
After bringing RSS subscription link into the sidebar this test case was constantly failing because of obvious reasons so I decided to change it a bit so [not it checks](After bringing RSS subscription link into the sidebar this test case ) that page includes `type="application/rss+xml"` string, which seems to be more relevant anyway.